nextflow.config

```diff
@@ -18,6 +18,12 @@ params {
     igenomes_base              = 's3://ngi-igenomes/igenomes'
     igenomes_ignore            = true
 
+    // Annotation options
+    run_annotation_tool             = 'prodigal'
+
+    // Prodigal Annotation
+    my_prodigal_f           = 'gff'
```

Comment:
`my_prodigal_f` is not very informative...

`annotation_prodigal_<whatever it refers to?>` might be better

Reply:
ok i just changed it to prodigal_output_format